Fix:

error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'ftp_transfer' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'gopher_transfer' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'http_transfer' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'isspace' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2000-08-08 19:36:32.000000000 -0500
+++ configure	2025-08-20 01:25:29.000000000 -0500
@@ -1977,6 +1977,7 @@
 #line 1978 "configure"
 #include "confdefs.h"
 #include <ctype.h>
+#include <stdlib.h>
 #define ISLOWER(c) ('a' <= (c) && (c) <= 'z')
 #define TOUPPER(c) (ISLOWER(c) ? 'A' + ((c) - 'a') : (c))
 #define XOR(e, f) (((e) && !(f)) || (!(e) && (f)))
--- http.c.orig	2000-08-08 19:33:30.000000000 -0500
+++ http.c	2025-08-20 01:28:22.000000000 -0500
@@ -2,6 +2,7 @@
 
 #include <config.h>
 
+#include <ctype.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
--- util.c.orig	2000-08-08 19:12:39.000000000 -0500
+++ util.c	2025-08-20 01:33:06.000000000 -0500
@@ -34,6 +34,9 @@
 #include <time.h>
 #include "url.h"
 #include "options.h"
+#include "gopher.h"
+#include "ftp.h"
+#include "http.h"
 
 
 char output_buf[BUFSIZ];
